plugins/woocommerce/includes/admin/settings/class-wc-settings-products.php

```diff
@@ -29,6 +29,7 @@ public function __construct() {
 		$this->label = __( 'Products', 'woocommerce' );
 
 		parent::__construct();
+		add_filter( 'woocommerce_admin_settings_sanitize_option_woocommerce_notify_no_stock_amount', 'absint' );
```

Comment:
A while ago we added a class to handle this kind of "one off" sanitization, which might be better suited given we've been trying to modernize the codebase a bit. The class is called `OptionSanitizer` and you can find it in `plugins/woocommerce/src/Internal/Settings/OptionSanitizer.php`.

In the constructor over there, you can add this same `add_filter()`.

Reply:
@jorgeatorres Filter moved, please give it another look.